reporter-web-app/src/components/AboutModal.js

```diff
@@ -46,7 +46,21 @@ SyntaxHighlighter.registerLanguage('yaml', yaml);
 const AboutModal = (props) => {
     const { webAppOrtResult } = props;
     const { repositoryConfiguration } = webAppOrtResult;
-    const { labels } = webAppOrtResult;
+    const {
```

Comment:
commit message: The sign off for "Marcel Bochtler" is duplicated. Is a sign off for "Stephanie Neubauer" missing as well?